WebHostLib/templates/supportedGames.html

```diff
@@ -21,6 +21,10 @@ <h2>{{ game_name }}</h2>
             {% elif world.web.settings_page %}
             <a href="{{ url_for("player_settings", game=game_name) }}">Settings Page</a>
             {% endif %}
+            {% if world.web.bug_report_page %}
+            <span class="link-spacer">|</span>
+            <a href="{{ world.web.bug_report_page }}">Bug Report</a>
```

Comment:
Should probably change text of anchor tag to a call to action like "Report a Bug" or "Report a Bug with this Game"